developer-docs-site/docs/concepts/base-gas.md

```diff
@@ -232,50 +234,69 @@ Payload gas is defined in [`transaction.rs`], which incorporates storage gas wit
-The same ratios apply among per-byte costs, with the effect that per-item costs are 2000 times higher than per-byte costs.
+Hence per-item operations cost 1000 times more than per-byte operations for both reads and creates, but only 60 times more for writes.
 
-In the absence of a legitimate economic incentive to deallocate from global storage (via either `move_from<T>()` or by removing from a table), this means that the most effective strategy for minimizing gas costs involves:
+Thus, in the absence of a legitimate economic incentive to deallocate from global storage (via either `move_from<T>()` or by removing from a table), the most effective storage gas optimization strategy is as follows:
 
-1. Minimizing per-item creations
-2. Tracking unused items and overwriting them, rather than creating new items, when possible
-3. Containing per-item writes to as few items as possible
-4. Reading, rather than writing, whenever possible
-5. Minimizing the number of bytes in any given operation, noting that per-item costs far outweigh optimizations at the per-byte level
+1. Minimize per-item creations
+2. Track unused items and overwrite them, rather than creating new items, when possible
+3. Contain per-item writes to as few items as possible
+4. Read, rather than write, whenever possible
+5. Minimize the number of bytes in all operations, especially writes
 
 ### Instruction gas
 
-As of the time of this writing, by far the most expensive instruction gas operation defined in [`instr.rs`] is a function call, which requires 1500 gas units (.15 octals), some 53 times less gas than a single per-item read in global storage.
-Loading a constant costs 650 gas units (.065 octals), borrow operations cost 500 units (0.05 octals), reading or writing to a reference costs 200 gas units (0.02 octals), and loading a `u128` on the stack costs 80 gas units (0.008 octals).
-Hence pass-by-value is less expensive than pass-by-reference for a primitive type like `u64`, but more expensive for a larger data structure like a 40-byte `struct`.
+As of the time of this writing, all instruction gas operations are multiplied by the `EXECUTION_GAS_MULTIPLIER` defined in [`gas_meter.rs`], which is set to 20.
+Hence the following representative operations assume gas costs as follows:
+
+| Operation                    | Gas units | Octals |
+|------------------------------|-----------|--------|
+| Table add/borrow/remove box  | 24,000    | 2.4    |
+| Function call                | 20,000    | 2      |
+| Load constant                | 13,000    | 1.3    |
+| Globally borrow              | 10,000    | 1      |
+| Read/write reference         | 4,000     | .4     |
+| Load `u128` on stack         | 1,600     | 0.16   |
+| Table box operation per byte | 200       | 0.02   |
```

Comment:
Same here. We should use external gas units instead of the internal ones. (The gas unit numbers should be divided by 10,000)

The cost in octa should also be recalculated following the example I gave above. 

We should also mention that these costs does not account for the actual storage costs: those will be measured separately.

Reply:
Per above, please see feb340f5cb1e3302168ed85173bc5668f27319e5.